### PR TITLE
Add abstract to the README

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
 
 ## Abstract
 
+Most methods for data-driven research (including Big Data, Data Science, and Digital Humanities) work primarily on text data or numbers.
+However, there is also a lot of information which is only available in printed books or newspapers.
+These information has to be first digitized and then further processed to extract the text or data.
+The main focus of the talk is optical character recognition (OCR).
+We will see the OCR workflow in general, discuss some OCR software, and how you can use them practically.
+Building such an infrastructure before the actual research analysis can start, may need a reasonable amount of time and resources, or also be a project itself.
+The Mannheim University Library has in this area some infrastructre projects which are briefly mentioned.
```
Comment (stweil) [inline README.md]: s/infrastructre/infrastructure/

File: README.md [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
 
-* **Speaker:** [Philipp Zumstein](https://www.bib.uni-mannheim.de/philipp-zumstein/) (<img src="https://assets-cdn.github.com/images/modules/logos_page/GitHub-Mark.png" height="20px"/> [@zuphilip](https://github.com/zuphilip), <img src="https://cdn.pixabay.com/photo/2014/04/03/11/53/twitter-312464_640.png?attachment"/ height="18px"> [@zuphilip](https://twitter.com/zuphilip))
-* **Venue:** Social Science Data Lab, MZES, Mannheim, March 15th, 2017
 
 
 ## Abstract
 
+Most methods for data-driven research (including Big Data, Data Science, and Digital Humanities) work primarily on text data or numbers.
+However, there is also a lot of information which is only available in printed books or newspapers.
+These information has to be first digitized and then further processed to extract the text or data.
```
Comment (stweil) [inline README.md]: _This_ information?

File: README.md [hunk 3]
```diff
@@ -1,13 +1,27 @@
...
+The main focus of the talk is optical character recognition (OCR).
```
Comment (stweil) [inline README.md]: I suggest _text recognition or, as it is traditionally called, optical character recognition (OCR)_.

File: README.md [hunk 4]
```diff
@@ -1,13 +1,27 @@
...
+We will see the OCR workflow in general, discuss some OCR software, and how you can use them practically.
```
Comment (stweil) [inline README.md]: Singular or plural? _software products_ => _you can use them_, or just _software_ => _you can use that software_

File: README.md [hunk 5]
```diff
@@ -1,13 +1,27 @@
...
+Building such an infrastructure before the actual research analysis can start, may need a reasonable amount of time and resources, or also be a project itself.
+The Mannheim University Library has in this area some infrastructre projects which are briefly mentioned.
+
+
+## Keywords
+
+* [optical character recoginition (OCR)](https://en.wikipedia.org/wiki/Optical_character_recognition)
```
Comment (stweil) [inline README.md]: s/recoginition/recognition/

File: README.md [hunk 6]
```diff
@@ -17,3 +31,8 @@
 
 ## Links
 
+* https://github.com/tmbdev/ocropy
+* https://github.com/tesseract-ocr/tesseract
+* http://digi.bib.uni-mannheim.de/aktienfuehrer/
```
Comment (stweil) [inline README.md]: Maybe s/http/https/

File: README.md [hunk 7]
```diff
@@ -17,3 +31,8 @@
...
+* https://digi.bib.uni-mannheim.de/periodika/reichsanzeiger//
```
Comment (stweil) [inline README.md]: One trailing slash is sufficient.

File: README.md (hunk 3 above)
Comment (zuphilip) [reply README.md]: I will leave this as it is.

(@stweil I know what you mean, but for me the term is still "optical character recognition", also the recognition nowadays does not work character by character. But even if you are using neural networks, then you have recognized the characters in the end, which can be also numbers besides text. Besides Wikipedia there is also the ACM classification which still uses traditional term: http://dl.acm.org/ccs/ccs.cfm?id=10010508&lid=0.10010405.10010497.10010504.10010508 )

**Comment by zuphilip** [comment]
@stweil Thank you very much for the review. I fixed all typos and improved the writing.

**Pull request merged**